builtin/logical/transit/path_sign_verify.go

```diff
@@ -226,6 +232,11 @@ derivation is enabled; currently only available with ed25519 keys.`,
 				Description: `Hash algorithm to use (POST URL parameter)`,
 			},
 
+			"mac_length": {
+				Type:        framework.TypeInt,
+				Description: `MAC length to use (POST body parameter). Valid values are:`,
```

Comment:
Where do these values get specified?

Reply:
They are processed solely on the enterprise side within `parseCmacVerifyInput` as an individual POST parameter or as a parameter within the `batch_input`. They just need to be defined here as we piggy back on the CE /verify api call.

Reply:
Got it.